com.reprezen.swagedit.openapi3/resources/com.reprezen.swagedit.openapi3.xml

```diff
@@ -13,5 +13,7 @@
 		<mapping pluginKey="colorTagPropertyPreferences" themeKey="foreground" />
 		<mapping pluginKey="colorFlowCharacterPreferences" themeKey="foreground" />
 		<mapping pluginKey="colorConstantPreferences" themeKey="method" />
+        <mapping pluginKey="AbstractTextEditor.Color.Foreground" themeKey="foreground"/>
```

Comment:
@tfesenko , small thing, but it looks like the indent level is not consistent. Maybe a tabs vs. spaces problem.